fdbserver/ptxn/test/TestTLogServer.actor.cpp

```diff
@@ -46,10 +66,26 @@ ACTOR Future<Void> startTLogServers(std::vector<Future<Void>>* actors,
 		Promise<Void> recovered;
 		tLogInitializations.emplace_back();
 		tLogInitializations.back().isPrimary = true;
+		tLogInitializations.back().storeType = KeyValueStoreType::MEMORY;
 		tLogInitializations.back().tlogGroups = pContext->tLogGroups;
 		UID tlogId = ptxn::test::randomUID();
 		UID workerId = ptxn::test::randomUID();
-		actors->push_back(ptxn::tLog(std::vector<std::pair<IKeyValueStore*, IDiskQueue*>>(),
+		StringRef fileVersionedLogDataPrefix = LiteralStringRef("log2-");
```

Comment:
New code prefers to use `"log2-"_sr` over `LiteralStringRef("log2-")`, i.e., using `_sr` operator.